parsers/cxx/cxx_parser_tokenizer.c

```diff
@@ -1327,12 +1335,12 @@ bool cxxParserParseNextToken(void)
 			}
 		} else {
 
-			const cppMacroInfo * pMacro = cppFindMacro(vStringValue(t->pszWord));
+			cppMacroInfo * pMacro = cppFindMacro(vStringValue(t->pszWord));
 
-			if(pMacro)
+			CXX_DEBUG_PRINT("Macro %s <%p> useCount: %d", vStringValue(t->pszWord),
```

Comment:
I'd move this debug statement inside the if, so it isn't printed at every token.

Reply:
I see. I will reflect the comment to this pull request.

Reply:
I changed as following to report overly used macros:
```
+#ifdef DEBUG
+                       if(pMacro && (pMacro->useCount >= CXX_PARSER_MAXIMUM_MACRO_USE_COUNT))
+                       {
+                               /* If the macro is overly used, report it here. */
+                               CXX_DEBUG_PRINT("Overly uesd macro %s <%p> useCount: %d (> %d)",
+                                                               vStringValue(t->pszWord),
+                                                               pMacro, pMacro? pMacro->useCount: -1,
+                                                               CXX_PARSER_MAXIMUM_MACRO_USE_COUNT);
+                       }
+#endif
```